MediaBrowser.Controller/SystemBackupService/BackupManifestDto.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace MediaBrowser.Controller.SystemBackupService;
+
+/// <summary>
+/// Manifest type for backups internal structure.
+/// </summary>
+public class BackupManifestDto
+{
+    /// <summary>
+    /// Gets or sets the jellyfin version this backup was created with.
+    /// </summary>
+    public required Version JellyfinVersion { get; set; }
+
+    /// <summary>
+    /// Gets or sets the backup engine version this backup was created with.
+    /// </summary>
+    public required Version BackupEngineVersion { get; set; }
+
+    /// <summary>
+    /// Gets or sets the date this backup was created with.
+    /// </summary>
+    public required DateTimeOffset DateOfCreation { get; set; }
```

Comment:
If we want to be consistent with other API models (e.g. baeitemdto)

```suggestion
    public required DateTimeOffset DateCreated { get; set; }
```